malaria-ios/ViewControllers/UserProfilePillTableViewCell.swift

```diff
@@ -0,0 +1,45 @@
+import UIKit
+import DoneToolbarSwift
+
+// MARK: - Table View Cell
+
+class UserProfilePillTableViewCell: UITableViewCell {
+  
+  @IBOutlet weak var pillName: UILabel!
+  @IBOutlet weak var quantityTextField: UITextField!
+  
+  weak var delegate : SaveRemainingPillsProtocol?
+  
+  var toolBar: ToolbarWithDone!
+  
+  func updateCellWithParameters(delegate: SaveRemainingPillsProtocol,
+                                name: String,
+                                remainingMedicine: String,
+                                indexPath: NSIndexPath) {
+    pillName.text = name
+    print(remainingMedicine)
+    quantityTextField.text = remainingMedicine
+    quantityTextField.delegate = self
+    quantityTextField.tag = indexPath.row
+    toolBar = ToolbarWithDone(viewsWithToolbar: [quantityTextField])
```

Comment:
I don't think it is easy but try to just set this value once (you are always creating a new object whenever the cell is requested to be rendered)